frontend/src/theme/components/Tile.ts

```diff
@@ -0,0 +1,67 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+const activeContainerStyle = {
+  bgColor: 'primary.200',
+  borderColor: 'transparent',
+  // NOTE: For the boxShadow styling, due to conflicts with the focus-visible package,
+  // the !important is required to display the boxShadow styling correctly.
+  boxShadow: '0 0 0 3px var(--chakra-colors-primary-500) !important',
```

Comment:
Should we add an `_hover` state for active that has `bg: 'primary.200'`  so hovering a selected tile won't have any jarring color changes?

Reply:
let me check back with pearly on this!

Reply:
clarified n changed!